nntrainer/src/tensor.cpp

```diff
@@ -579,12 +477,8 @@ Tensor Tensor::dot(Tensor const &m, bool trans, bool trans_m) const {
   float *rdata = result.getData();
   const float alpha = 1.0f;
   const float beta = 0.0f;
-#ifdef USE_BLAS
-  enum CBLAS_TRANSPOSE transA = trans ? CblasTrans : CblasNoTrans;
-  enum CBLAS_TRANSPOSE transB = trans_m ? CblasTrans : CblasNoTrans;
-  cblas_sgemm(CblasRowMajor, transA, transB, M, N, K, alpha, data, lda, mdata,
-              ldb, beta, rdata, ldc);
-#elif USE_CUBLAS
+
+#ifdef USE_CUBLAS
```

Comment:
with this CUBLAS has a higher priority than BLAS.
Can we list the order of priorities somewhere with this PR?

Reply:
I think this could go to blas_interface.cpp like gemm. I will do another PR.
The order should go.. blas, cublas, and withouth. I guess.

Reply:
Then yes, please make another PR, and set the order, and add to the documentation appropriately as well.